### DISCUSSION TITLE
[v4.1] Validator folder structure

### DISCUSSION THREAD

**Discussion by Christianrcds** [adonisjs/core#2472] [Help] [answered]
Hello everybody,

I want to create a folder for my validators so i can organize my structure in a better way  inside the folder "Validators", but i don't know how to route this.  
Example: 
`Route.post().validator(?)`

**Comment by thetutlage** [comment, marked as answer]
Which version of the framework

**Reply by Christianrcds** [reply]
It's using the 4.1.0 version

**Reply by artem-a** [reply]
Hi @Christianrcds you can create a resources structure 

the folders structure
![image](https://user-images.githubusercontent.com/3863679/114234278-33957680-9987-11eb-96d6-265567531dff.png)

using inside the routes file
![image](https://user-images.githubusercontent.com/3863679/114234373-558ef900-9987-11eb-83e4-dd333fa4e6f8.png)

**Reply by Christianrcds** [reply]
Nice, it's really simple. Thanks @artem-a, worked just fine!
